src/hipscat/io/file_io/file_io.py

```diff
@@ -37,6 +37,22 @@ def make_directory(
     file_system.makedirs(file_pointer, exist_ok=exist_ok)
 
 
+def handle_pandas_storage_options(
+    storage_options: Union[Dict[Any, Any], None]
+) -> Union[Dict[Any, Any], None]:
+    """Handle storage options for pandas read/write methods."""
```

Comment:
Add a link to the pandas documentation that you gave in the bug. That would help future developers understand why this method is needed.